components/atom/button/README.md

```diff
@@ -46,5 +46,24 @@ return (<div>
 
 ```
 
+### Rendering a link
+When `link` property is passed, the component will render an html link.
+
+```js
+import Button from '@schibstedspain/sui-atom-button'
+
+return (
+  <Button link url='http://www.schibsted.com/'>Link</Button>
```

Comment:
Why not unify props in just one? E.g.: `<Button link='http://www.schibsted.com/'>Link</Button>` So that, we would use just one string prop, instead of a string and a boolean.

Reply:
+1

Reply:
Agree with Kiko,
But, on the other hande, knowing that a target props can be applied, we may do 

```html
<Button link href='http://www.schibsted.com/'>Link</Button>
<Button link href='http://www.schibsted.com/' target='_blank'>Link</Button>
```

`link` prop would just mean "convert to link" and then you can use whatever `<a>` specific props.

Reply:
well seen! done!

Reply:
ok, i get your point @davidbarna. then, what if we send an object?
`<Button link={ href: 'http://www.schibsted.com/', target:'_blank'}>Link</Button>`

Reply:
We are trying to be as HTML standards compliant as possible. I really think if should be defined by props.

To me, the best API would be either:

```js
<Button link href='http://www.schibsted.com/' target='_blank'>Link</Button>
````
or 

```js
import {Link} from 'sui-atom-button'
<Link href='http://www.schibsted.com/' target='_blank'>Link</Link>
````

Reply:
ok, take a look @davidbarna. developed like so:

```html
<Button link href='http://www.schibsted.com/' target='_blank'>Link</Button>
```